extruct/dublincore.py

```diff
@@ -143,8 +141,8 @@ def populate_results(node, main_attrib):
         namespaces_nodes = document.xpath('//link[contains(@rel,"schema")]')
         namespaces = {}
         for i in namespaces_nodes:
-            if i.attrib['href'] in _URL_NAMESPACES:
-                namespaces.update({re.sub(r"schema\.", "", i.attrib['rel']): i.attrib['href']})
+            if strip_html5_whitespace(i.attrib['href']) in _URL_NAMESPACES:
```

Comment:
would you mind adding a local variable for href? It'd allow to save a small bit of computation, and also make code a bit more readable (line would be shorter)

Reply:
@kmike , thanks for pointing it out :pray: , I just modified it.